### PR TITLE
Bfp market natspec refactor

### PR DESCRIPTION
**Pull request by davidvuong**

### CODE REVIEW

File: markets/bfp-market/storage.dump.sol
Comment (davidvuong) [inline markets/bfp-market/storage.dump.sol:1]: Something is very wrong with `storage.dump.sol`. After compiling contracts it has decided to wipe away recent changes and replace with old structs.

File: markets/bfp-market/contracts/interfaces/ILiquidationModule.sol [hunk 1]
```diff
@@ -71,31 +94,36 @@ interface ILiquidationModule is IBasePerpMarket {
...
+    /// @param accountId Account of margin to query against
+    /// @param marketId Market of margin to query against
+    /// @return isMarginLiquidatable True if margin can be liquidated, false otherwise
     function isMarginLiquidatable(uint128 accountId, uint128 marketId) external view returns (bool);
 
-    /**
-     * @notice Returns the IM (initial maintenance) and MM (maintenance margin) for a given account, market, and sizeDelta.
-     * Specify a sizeDelta of 0 for the current IM/MM of an existing position.
-     */
+    /// @notice Returns the IM (initial maintenance) and MM (maintenance margin) for a given account, market, and sizeDelta.
```
Comment (0xjocke) [inline markets/bfp-market/contracts/interfaces/ILiquidationModule.sol:112]:
```suggestion
    /// @notice Returns the IM (initial margin) and MM (maintenance margin) for a given account, market, and sizeDelta.
```

File: markets/bfp-market/storage.dump.sol
Comment (davidvuong) [reply markets/bfp-market/storage.dump.sol:1]: Fixed after clearing cache and a plethora of other things to regenerate with latest contracts.

File: markets/bfp-market/contracts/interfaces/IPerpAccountModule.sol [hunk 2]
```diff
@@ -5,103 +5,108 @@ interface IPerpAccountModule {
...
+    /// @notice Merges two accounts, combining `fromId` into `toId` for `marketId`. Merging accounts will realize the
+    ///         position of account `toId` in addition to transferring collateral and size from one to the other. It's
+    ///         on the caller to burn the perp account NFT post merge.
+    ///         Additionally, this fn requires that account `fromId` must have just been settled, implying account
+    ///         merge operation can only be performed in the same block as settlement via multicalls on settlement or
+    ///         indirectly settlement hooks.
+    /// @param fromId Account to merge from
+    /// @param toId Account to merge into
+    /// @param marketId Market to perform the account merge
+    /// @dev Account permmisions in the `fromId` account are _not_ be transferred and that we only allow
```
Comment (0xjocke) [inline markets/bfp-market/contracts/interfaces/IPerpAccountModule.sol:99]:
```suggestion
    /// @dev Account permissions in the `fromId` account are _not_ transferred and we only allow
```

**Review by 0xjocke — APPROVED** [review]
Two minor things, otherwise looks good. I guess we need to figure out the storage dump too though

**Comment by davidvuong** [comment]
> Two minor things, otherwise looks good. I guess we need to figure out the storage dump too though

Got it to regenerate correctly 😑

**Pull request merged**